vpr/src/base/vpr_types.h

```diff
@@ -1199,6 +1199,9 @@ struct t_placer_opts {
     int floorplan_num_horizontal_partitions;
     int floorplan_num_vertical_partitions;
 
+    int placer_debug_block;
```

Comment:
Please add a brief comment for each of these; it can refer to the command line parser help for more detail if you want to avoid duplication.